### DISCUSSION TITLE
Dedicated Server

### DISCUSSION THREAD

**Discussion by Silentt00** [official-antistasi-community/A3-Antistasi#2648] [Q&A] [answered]
I am new to running a dedicated server with mods. after about a day and a half of figuring it out I am finally showing the mod in #missions on the game. However I run into an error saying I cannot "play/edit this mod. it relies on downloadable content that has been deleted. A3A maps" I also do not have a picture or description when I click antistasi.

Any help would be appreciated

**Comment by Rangifers** [comment]
You need to move the key file(s) from any mod(s) you are using into your servers key folder.

**Comment by riwarek** [comment]
Where are saves of antistasi currently? Are they in dedicated_server\Users\win_profile\win_profile.vars.Arma3Profile?
What for is AntistasiCommunity.vars?
And finally, how can I move saves between dedicated servers?

**Reply by HakonRydland** [reply]
the old save system used the profile vars file, if you use the new file option it will store it in the AntistasiCommunity vars file instead

**Reply by riwarek** [reply]
Then as I understand I should simply copy antistasiCommunity.vars to other machine. Am I right? Of course I don't consider now mods, keys and other stuff.

**Comment by HakonRydland** [comment, marked as answer]
@Silentt00 for a dedicated server with antsitasi you need to load the mod on both the server and the clients, in addition if you have verify signatured on for the server you need to copy the keys from any mods you load into the servers key folder

**Comment by Chris-EduardDunareanu** [comment]
Hello i have a problem, i have antistasi installed on both host and client, copied the keys into the server key folder but the error:  You cannot "play/edit this mod. it relies on downloadable content that has been deleted. A3A maps" still appers.
I am using Faster and steamcmd, arma 3 server.

**Reply by jaj22** [reply]
Provide the RPT file from the server.

**Reply by Chris-EduardDunareanu** [reply]
https://pastebin.com/UcsDJBfc here is the RPT File,

![image](https://github.com/official-antistasi-community/A3-Antistasi/assets/38765738/2a575267-c0d6-4073-8fc7-ea6802d7aa8b)
this is the arma 3 server directory

**Reply by jaj22** [reply]
There's no attempt to start a mission in this RPT, but apparently the server is running version 3.0 and not 3.2. Delete all the mod files and redownload. Then double-check that the mod's files are identical to the mod on the client.

**Reply by Chris-EduardDunareanu** [reply]
worked! Thanks
